tools/proofers/codepoint_validator.js

```diff
@@ -34,15 +34,17 @@ $(function () {
         }
     });
 
-    var badPattern = new RegExp("[^" + charClass + "]", "ug");
+    var badClass = "[^" + charClass + "]";
+    var badPattern = new RegExp(badClass, "ug");
+    var badPattern0 = new RegExp(badClass, "u");
```

Comment:
I *really* dislike variables with numbers at the end. Can we rename these to what they are?
```
badPatternGlobal
badPattern
```

Reply:
Sorry about that.